recipe/meta.yaml

```diff
@@ -21,7 +21,6 @@ requirements:
   build:
     - python
     - cmake
-    - ninja
```

Comment:
@jakirkham, I can leave it if you want to test, I could not make it work properly